# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/moz.build
# Commit: dbdca4661a35
# Full Hash: dbdca4661a35c55d343cb76fd10366cdbd1e4c09
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-18 09:51:31
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - PFetch protocol declaration and implementation. r=dom-worker-reviewers,jesup
#   
#   Depends on D138813
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142436
# ==============================================================================

diff -r ea27cd66fefd -r dbdca4661a35 dom/fetch/moz.build
--- a/dom/fetch/moz.build	Tue Jan 17 15:13:44 2023 +0000
+++ b/dom/fetch/moz.build	Tue Jan 17 15:13:45 2023 +0000
@@ -11,9 +11,11 @@
     "BodyExtractor.h",
     "ChannelInfo.h",
     "Fetch.h",
+    "FetchChild.h",
     "FetchDriver.h",
     "FetchIPCTypes.h",
     "FetchObserver.h",
+    "FetchParent.h",
     "FetchService.h",
     "FetchStreamReader.h",
     "FetchStreamUtils.h",
@@ -30,8 +32,10 @@
     "BodyExtractor.cpp",
     "ChannelInfo.cpp",
     "Fetch.cpp",
+    "FetchChild.cpp",
     "FetchDriver.cpp",
     "FetchObserver.cpp",
+    "FetchParent.cpp",
     "FetchService.cpp",
     "FetchStreamReader.cpp",
     "FetchStreamUtils.cpp",
@@ -46,6 +50,7 @@
 
 IPDL_SOURCES += [
     "FetchTypes.ipdlh",
+    "PFetch.ipdl",
 ]
 
 LOCAL_INCLUDES += [